package.json

```diff
@@ -35,7 +35,7 @@
     "express": "^4.13.4",
     "juttle": "^0.4.2",
     "juttle-aws-adapter": "^0.1.3",
-    "juttle-cloudwatch-adapter": "^0.1.2",
+    "juttle-cloudwatch-adapter": "^0.2.0",
```

Comment:
yes "This PR also bumps the juttle-cloudwatch-adapter to version 0.2.0. I
can pull that out into a separate PR if necessary."